package.json

```diff
@@ -8,7 +8,7 @@
     "dev": "concurrently 'netlify dev' 'yarn netlify-cms-proxy-server'",
     "prebuild": "rimraf dist",
     "build": "yarn typecheck; vite build && workbox generateSW workbox-config.js",
-    "lint": "eslint src admin --ext .ts,.tsx",
+    "lint": "eslint --fix src admin --ext .ts,.tsx",
```

Comment:
Do we want to update this script? `--fix` is already being passed in `lint-staged`

Reply:
Oh yes you're right. We can remove this from here. I'll update PR